src/ex2/ItemManager.js

```diff
@@ -0,0 +1,54 @@
+class ItemManager {
```

Comment:
these seems to be leftovers from last exercise, you should merge the previous PR and then update this one from master so these changes will be ignored